src/Sylius/Behat/Context/Ui/Admin/ManagingShipmentsContext.php

```diff
@@ -96,6 +106,14 @@ public function iFilter(): void
         $this->indexPage->filter();
     }
 
+    /**
+     * @When I view the shipment of the order :order
```

Comment:
```suggestion
     * @When I view the first shipment of the order :order
```
or only with some assertion